vm/src/stdlib/ast.rs

```diff
@@ -638,7 +638,7 @@ pub fn make_module(vm: &VirtualMachine) -> PyObjectRef {
     let ctx = &vm.ctx;
 
```

Comment:
I think you missed `"ast" -> "_ast"` in `create_node` and `AstNode::class` up above.

Reply:
Nice catch! See line 24. It might make sense to declare a global constant at the top of this file named `const module_name = "_ast";` and use it where appropriate.